NEWS.md

```diff
@@ -4,6 +4,15 @@ MXNet Change Log
 ### Deprecations
 - An incorrect [usage](https://github.com/apache/incubator-mxnet/issues/11091) of `save_params` was advertised in the gluon book which led to MXNet users depending on the incorrect usage and developing a hack around it. A change was made to the internal structure of the `.params` file saved by `save_params` to resolve a bug. This led to user scripts with the above mentioned hack to break. To fix this, `save_params` and `load_params` APIs have been reverted to previous format and marked as deprecated. New APIs: `save_parameters` and `load_parameters` have been added for the new format. All scripts to save and load parameters for a Gluon model should now use the new API for `save_parameters` and `load_parameters`. If your model is hybridizable and you want to export a serialized structure of the model as well as parameters you need to use the `export` API and the newly added `imports` API instead of `save_params` and `load_params` API. For more details, Please see: [issue](https://github.com/apache/incubator-mxnet/issues/11091), [PR](https://github.com/apache/incubator-mxnet/pull/11127).
 
+### User Code Changes
```

Comment:
Suggest to move this section on the top of this release.

Reply:
will do ..

Reply:
actually the deprecation section gives some background before introducing the user code changes section. I am not sure if deprecation under user code changes would make sense. I can remove the lines starting at "If your model is hybridizable" to make it more clear. What do you think ?